.pre-commit-config.yaml

```diff
@@ -22,7 +22,13 @@ repos:
     rev: v1.6.1
     hooks:
       - id: mypy
+        additional_dependencies: [
+          "elasticsearch[async]==8.6.1",
+          "elastic-transport==8.4.1",
+          "types-tabulate",
```

Comment:
Do we want to add a version here to match the one we're using?

Reply:
You're right, https://github.com/elastic/rally/pull/1859/commits/30849e6fe0450e166331d0769e12aac4967039a4.